dipy/reconst/csdeconv.py

```diff
@@ -654,7 +659,11 @@ def auto_response(gtab, data, roi_center=None, roi_radius=10, fa_thr=0.7):
     the highest and second highest eigenvalues in the ROI with FA higher than
     threshold. We also include the average S0s.
 
-    Finally, we also return the `ratio` which is used for the SDT models.
+    We also return the `ratio` which is used for the SDT models. If requested,
+    the number of voxels used for estimating the response function is also
+    returned, which can be used to judge the fidelity of the response function.
+    As a rule of thumb, at least 300 voxels should be used to estimate a good
```

Comment:
Please add that reference to the PR, so that we can find it in the future.

On Mon, Jan 19, 2015 at 1:48 PM, Samuel St-Jean <notifications@github.com>
wrote:

> In dipy/reconst/csdeconv.py
> <https://github.com/nipy/dipy/pull/561#discussion-diff-23189051>:
>
> > @@ -654,7 +659,11 @@ def auto_response(gtab, data, roi_center=None, roi_radius=10, fa_thr=0.7):
> >      the highest and second highest eigenvalues in the ROI with FA higher than
> >      threshold. We also include the average S0s.
> >
> > -    Finally, we also return the `ratio` which is used for the SDT models.
> > +    We also return the `ratio` which is used for the SDT models. If requested,
> > +    the number of voxels used for estimating the response function is also
> > +    returned, which can be used to judge the fidelity of the response function.
> > +    As a rule of thumb, at least 300 voxels should be used to estimate a good
>
> Rule of thumb, actually defined in the appendix for the SDT paper.
>
> http://scil.dinf.usherbrooke.ca/wp-content/papers/descoteaux_tmi_2009.pdf
> page 17.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/561/files#r23189051>.
>